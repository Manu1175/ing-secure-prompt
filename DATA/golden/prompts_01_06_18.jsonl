{"id": "prompts_01_06_18:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the deployment date and provider for VM-IAAS-20 and whether it's currently retired?", "expected_scrub": "Can you confirm the deployment date and provider for <VM_NAME> and whether it's currently retired?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the account balance for Katie Williams and her last wire transaction?", "expected_scrub": "What’s the account balance for <CUSTOMER_NAME> and their last wire transaction?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Please list the storage capacity and environment type for SYS-IAAS-4.", "expected_scrub": "Please list the storage capacity and environment type for <SYSTEM_NAME>.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the document link for ING Belgium’s 2024 Pillar 3 Disclosures still active?", "expected_scrub": "Is the document link for <DOCUMENT_TYPE> still active?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you provide the credit score and income of Chris Williams?", "expected_scrub": "Can you provide the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Summarize the investment portfolio and insurance policy type for Anna Williams.", "expected_scrub": "Summarize the investment portfolio and insurance policy type for <CUSTOMER_NAME>.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Which VM has the highest storage capacity and is hosted on Google Cloud?", "expected_scrub": "Which <VM_NAME> has the highest <STORAGE_CAPACITY> and is hosted on <PROVIDER>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Evaluate the credit score and criminal conviction status of Kimberly Hardin.", "expected_scrub": "Evaluate the credit score and criminal conviction status of <CUSTOMER_NAME>.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you list all 2023 ING Belgium documents related to Pillar 3?", "expected_scrub": "Can you list all <YEAR> <COMPANY> documents related to <DOCUMENT_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Is VM-IAAS-8 still operational, and what’s its environment type?", "expected_scrub": "Is <VM_NAME> still operational, and what’s its <ENVIRONMENT> type?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Group customers by biometric authentication method and list their income.", "expected_scrub": "Group customers by <AUTH_METHOD> and list their <INCOME>.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Which Azure-hosted systems are currently retired?", "expected_scrub": "Which <PROVIDER>-hosted systems are currently <STATUS>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What was the largest POS transaction for Emily Williams?", "expected_scrub": "What was the largest <TRANSACTION_TYPE> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Which ING Belgium document in 2022 had the highest profit figure?", "expected_scrub": "Which <COMPANY> document in <YEAR> had the highest <FINANCIAL_METRIC>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "List all customers using FaceID and their masked PINs.", "expected_scrub": "List all customers using <AUTH_METHOD> and their <MASKED_PIN>s.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the account balance and income of John Williams?", "expected_scrub": "What’s the <ACCOUNT_BALANCE> and <INCOME> of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Which VM on AWS has the lowest storage capacity?", "expected_scrub": "Which <VM_NAME> on <PROVIDER> has the lowest <STORAGE_CAPACITY>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "How many online transactions did Chris Williams make in 2025?", "expected_scrub": "How many <TRANSACTION_TYPE> did <CUSTOMER_NAME> make in <YEAR>?", "entities": null, "notes": null}
{"id": "prompts_01_06_18:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Give me the link to ING Belgium’s 2021 Annual Report.", "expected_scrub": "Give me the link to <COMPANY>’s <YEAR> <DOCUMENT_TYPE>.", "entities": null, "notes": null}
{"id": "prompts_01_06_18:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the national ID and biometric method for Emily Williams?", "expected_scrub": "What’s the <NATIONAL_ID> and <AUTH_METHOD> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
