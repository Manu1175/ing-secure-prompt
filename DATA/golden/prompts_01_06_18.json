[
  {
    "text": "Can you confirm the deployment date and provider for VM-IAAS-20 and whether it's currently retired?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the account balance for Katie Williams and her last wire transaction?",
    "labels": []
  },
  {
    "text": "Please list the storage capacity and environment type for SYS-IAAS-4.",
    "labels": []
  },
  {
    "text": "Is the document link for ING Belgium’s 2024 Pillar 3 Disclosures still active?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Can you provide the credit score and income of Chris Williams?",
    "labels": []
  },
  {
    "text": "Summarize the investment portfolio and insurance policy type for Anna Williams.",
    "labels": []
  },
  {
    "text": "Which VM has the highest storage capacity and is hosted on Google Cloud?",
    "labels": []
  },
  {
    "text": "Evaluate the credit score and criminal conviction status of Kimberly Hardin.",
    "labels": []
  },
  {
    "text": "Can you list all 2023 ING Belgium documents related to Pillar 3?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Is VM-IAAS-8 still operational, and what’s its environment type?",
    "labels": []
  },
  {
    "text": "Group customers by biometric authentication method and list their income.",
    "labels": []
  },
  {
    "text": "Which Azure-hosted systems are currently retired?",
    "labels": []
  },
  {
    "text": "What was the largest POS transaction for Emily Williams?",
    "labels": []
  },
  {
    "text": "Which ING Belgium document in 2022 had the highest profit figure?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "List all customers using FaceID and their masked PINs.",
    "labels": []
  },
  {
    "text": "What’s the account balance and income of John Williams?",
    "labels": []
  },
  {
    "text": "Which VM on AWS has the lowest storage capacity?",
    "labels": []
  },
  {
    "text": "How many online transactions did Chris Williams make in 2025?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Give me the link to ING Belgium’s 2021 Annual Report.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the national ID and biometric method for Emily Williams?",
    "labels": []
  }
]